src/main.cpp

```diff
@@ -108,6 +112,20 @@ namespace
         throw std::invalid_argument{"Please choose one of the accepted VCF fileformat versions"};
     }
 
+    std::string get_output_path(const std::string &outdir, const std::string &file_path)
+    {
+        if (outdir == "") {
+            return file_path;
+        }
+        
+        boost::filesystem::path file_boost_path{file_path};
+        boost::filesystem::path outdir_boost_path{outdir};
+        
+        outdir_boost_path /= file_boost_path.filename();
+        
+        return outdir_boost_path.string();
+    }
+
     std::vector<std::unique_ptr<ebi::vcf::ReportWriter>> get_outputs(std::string const &output_str, std::string const &input) {
```

Comment:
The warning message this function displays should refer to reports and the "-r/--report" argument now.